src/components/PromptForm/AutoCompleteInput.tsx

```diff
@@ -0,0 +1,53 @@
+import { Popover, PopoverTrigger, PopoverContent, PopoverBody } from "@chakra-ui/react";
+import { Box, Text } from "@chakra-ui/react";
+
+const AutoCompleteProps = ({
```

Comment:
```suggestion
const AutoCompleteInput = ({
```

This should be named `AutoCompleteInput`.